src/richie/apps/courses/models/organization.py

```diff
@@ -241,4 +241,39 @@ def __str__(self):
         )
 
 
+class OrganizationsForCategoryPluginModel(PagePluginMixin, CMSPlugin):
+    """
+    Handle the relation between a OrganizationsForCategoryPlugin plugin and its Category
+    instance.
+    """
+
+    page = models.ForeignKey(
+        Page,
+        on_delete=models.CASCADE,
+        related_name="organizations_for_category_plugins",
+        limit_choices_to={
+            "publisher_is_draft": True,  # plugins work with draft instances
+            "category__isnull": False,  # limit to pages linked to a category object
+        },
+    )
+    variant = models.CharField(
+        _("variant"),
+        max_length=50,
+        choices=defaults.ORGANIZATION_GLIMPSE_VARIANT_CHOICES,
+        help_text=_("Optional glimpse variant for a custom look."),
+        blank=True,
+        null=True,
+    )
+
+    class Meta:
+        db_table = "richie_organizations_for_category_plugin"
+        verbose_name = _("organizations for category plugin")
```

Comment:
```suggestion
        verbose_name = _("organizations by category plugin")
```

IMO it's easier to understand this way, eg. when you first encounter the plugin.

We could keep the name in the code unchanged as that's more of a bother to change

Reply:
Not a problem to change it in the code as it is not yet merged... So ok I will change it everywhere.